src/remote-config/remote-config-types.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface RemoteConfigParameter {
```

Comment:
Fix indentation.

And why are these in a separate file/module?

Reply:
I was following the `messaging-types.ts` as an example. I think it will make more sense to move these to `remote-config.ts`. Will update the PR!

Reply:
Updated.